server/data/draftReportClient.test.ts

```diff
@@ -121,3 +122,31 @@ test('getInvolvedStaff', async () => {
     values: [1, 'user-1', ReportStatus.IN_PROGRESS.value],
   })
 })
+
+test('getDuplicateReports', () => {
+  const startDate = moment('2021-07-13')
+  const endDate = moment('2021-07-13')
+
+  draftReportClient.getDuplicateReports(1, [startDate, endDate])
+  expect(query).toBeCalledWith({
+    text: `select r.incident_date date, r.form_response form, r.reporter_name reporter, r.status status
+              from v_report r where r.booking_id >= $1
+              and r.incident_date >= $2
+              and r.incident_date <= $3`,
+    values: [1, startDate, endDate],
+  })
+})
+test('deleteReport', () => {
```

Comment:
meh.. don't need to change but would be good to try to keep formatting consistent - other tests in this file have new lines between them. Ideally eslint would enforce this uniformity

Reply:
ok